users/models.py

```diff
@@ -15,7 +15,7 @@ class Professor(models.Model):
     first_name = models.CharField('First Name', max_length= 120)
     last_name = models.CharField('Last Name', max_length= 120)
     email = models.EmailField(default= 'user@odu.edu')
-    is_professor = models.BooleanField(default= False)
+    is_professor = models.BooleanField(default= True)
```

Comment:
Again, slipped by me earlier. What does is_professor do? If they aren't a professor, they should be in the student table, or this table should be renamed to faculty.